HW34/testts.ts

```diff
@@ -0,0 +1,97 @@
+// 1
+type faculty = {

```

Comment:
usually, types names start with a capital letter
it helps to separate types and variables